.yarnrc

```diff
@@ -0,0 +1 @@
+--ignore-engines true
```

Comment:
Looks like we don't need to do this as Yarn 2 skips engine validation 👍

https://github.com/yarnpkg/yarn/issues/8072#issuecomment-753670472

Reply:
I removed the file. 

I'm not super familiar with Yarn (yet), but it looks like we need to migrate our project over to yarn 2: https://yarnpkg.com/getting-started/migration